static.md

```diff
@@ -0,0 +1,92 @@
+# Heroku Buildpack: Nginx for static sites
+
+Operates as an HTTP server for websites and single page apps.
+
+## Usage
+
+### 1. Add Buildpack
+
+Add this buildpack branch to an app, as the last buildpack:
+```bash
+heroku buildpacks:add --app APP_NAME https://github.com/heroku/heroku-buildpack-nginx.git#improve-herokuness
+```
+(This should be updated to `heroku-community/nginx` once merged.)
+
+### 2. Create Procfile
+
+Create the `Procfile` in the root of your app, containing,
+```
+web: bin/start-nginx-static
+```
+
+### 3. Create config file
```

Comment:
Might be nice to make this step optional. The default config in the buildpack is somewhat customizable with simple env vars. If the application doesn't have a config/nginx-static.conf.erb, use the one embedded in the buildpack?

Reply:
That's unfortunately not possible, because the original default, zero-config setup is very different than this one: different sub-process management (start-up script starts multiple background processes instead of Nginx directly), different logging (to files instead of stdout), different default always-HTTPS config (not set instead of enabled). It would break backwards compatibility for folks already using this buildpack.

The only way to make copying the Nginx config optional, is to make a new buildpack for Static sites, which I am trying to avoid at this stage. This is incremental, backward-compatible improvement.